src/plugins/discover/public/application/main/components/no_results/no_results_suggestions/assets/no_results_illustration.tsx

```diff
@@ -12,8 +12,8 @@ import React from 'react';
 export const NoResultsIllustration = () => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
-    width="226"
-    height="166"
+    width="290"
+    height="206.14"
```

Comment:
I took it from figma design. Updated the height now.

Reply:
Oops, I didn't mean to imply that I thought the dimensions were necessarily wrong. It was just surprising to see a decimal like `.14` or `.01` for a pixel value since I wouldn't think it adds any value, but it was just a nit and not really something important. I think whichever dimensions we use are fine in the end.